nomad/parts/plugins/x-nomad.py

```diff
@@ -0,0 +1,49 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2015, 2016 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""This is a modifyied version of snapcraft's go and make plugins.
+
+It uses code from  the go plugin to pull, and the make plugin to build and
+install.
+
+"""
+import os
+import shutil
+
+from snapcraft import BasePlugin
+from snapcraft.plugins import go
+
+
+class NomadPlugin(go.GoPlugin):
+
+    def build(self):
+        BasePlugin.build(self)
```

Comment:
@sergiusens @kyrofa: what about this one? ^
The go plugin is not easy to extend. So what is the recommendation here? Copy the full plugin and inherit from BasePlugin, or do this awful inheritance trick to skip the parent? We could also do multiple inheritance, but I think that's a bigger mess.